### ISSUE TITLE
Update Documentation

### ISSUE THREAD

**Issue by golmek** [golemfactory/golem-docs#199]
**Prerequisite:** https://github.com/golemfactory/yagna/issues/3110

Regarding Onboarding portal release:

- Those two pages can be merged together and moved upward in documents structure (not to be embedded into Python/JavaScript:
   - https://docs.golem.network/docs/creators/javascript/guides/switching-to-mainnet
   - https://docs.golem.network/docs/creators/python/guides/switching-to-mainnet

- To be updated:
   - https://docs.golem.network/docs/creators/python/examples/tools/managing-golem-wallet#enable-the-mainnet-account
   - https://docs.golem.network/docs/golem/payments/golem-token-conversion#mainnet-polygon-glm-conversion (NOT IN THE SCOPE)

- To be verified:
  - https://docs.golem.network/docs/golem/payments (NOT IN SCOPE)

Section: funding golem wallet
